Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,103 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Host network mode
+#################
+
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium Gateway API Gateway directly on the
+host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external loadbalancer,
+...).
+
+.. note::
+    * Enabling the Cilium Gateway API host network mode automatically disables the LoadBalancer type service mode. They are mutually exclusive.
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+
+The host network mode can be enabled via Helm flag:
+
+.. code-block:: yaml
+
+    gatewayAPI:
+      enabled: true
+      hostNetwork:
+        enabled: true
+
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
```

Comment:
Is there only one host network port per Gateway resource? If so, can we reword to make this more clear?

```suggestion
Once enabled, the host network port for a ``Gateway`` can be specified via ``spec.listeners.port``.
The port must be unique per ``Gateway``
```